index.html

```diff
@@ -0,0 +1,98 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, shrink-to-fit=no"
+    />
+    <link rel="stylesheet" href="style.css" />
+    <title>Product Landing page</title>
+  </head>
+  <body id="body">
+    <hearder id="header">
+      <div class="b-nav"><a href="#body">
+        <img  src="motorcycle.png" alt="motorcycle" id="header-img" />
+      </a>
+        <h2>Motorcycle</h2>
+      </div>
+      <div class="nav">
+
+        <nav id="nav-bar">
+          <a href="#catagories" class="nav-link">Catagories</a>
+          <a href="#pricing"class="nav-link" >Pricing</a>
+          <a href="#contact"class="nav-link" >Contact Us</a>
+        </nav>
+      </div>
+    </hearder>
+
+    <div id="video1">
+      <!-- <video  controls id="video">
+        <source src="./video.mp4" type="video/mp4" />
+      </video> -->
+      <iframe  id="video" width="100%" height="100%" src="https://www.youtube.com/embed/7g1R_LD0yyY">
+      </iframe>
+    </div>
```

Comment:
Please use HTML5 semantics tags. By using semantic tags, you help the browser understand the meaning of the content, and also it will increase your SEO score.